lib/dialects/abstract/query-generator.js

```diff
@@ -489,9 +483,7 @@ class QueryGenerator {
       - rawTablename, the name of the table, without schema. Used to create the name of the index
    @private
   */
-  addIndexQuery(tableName, attributes, options, rawTablename) {
-    options = options || {};
-
+  addIndexQuery(tableName, attributes, options = {}, rawTablename) {
```

Comment:
Just a suggestion, it's better if optionals parameters are explicits: `rawTablename = null`